### PR TITLE
Changed Promotion into BookDiscount, convergence fix, added WireMock …

### PR DESCRIPTION
**Pull request by Tymoteuszauke**
…for stubs

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -94,8 +95,18 @@
...
+                <exclusion>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-lang3</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.6</version>
```
Comment (Dzionek95) [inline pom.xml:108]: would be better to have it in properties

**Review by Dzionek95 — APPROVED** [review]

File: models/src/main/java/view/book_discount/BookAuthorView.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package view.promotion;
+package view.book_discount;
```
Comment (mprtcz) [inline models/src/main/java/view/book_discount/BookAuthorView.java:1]: package names should not contain any special characters, only letters :hankey:

File: restbot/pom.xml [hunk 3]
```diff
@@ -18,21 +18,48 @@
...
     <properties>
+        <!--General properties-->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>1.8</java.version>
-        <daoVersion>1.0.0</daoVersion>
-        <flywayVersion>4.2.0</flywayVersion>
+        <!--Dependency properties-->
+        <modelsVersion>1.0-SNAPSHOT</modelsVersion>
+        <wireMockVersion>2.7.1</wireMockVersion>
```
Comment (patkkka) [inline restbot/pom.xml:27]: to properties!

File: restbot/pom.xml [hunk 4]
```diff
@@ -18,21 +18,48 @@
...
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
+                    <artifactId>guava</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>23.0</version>
```
Comment (patkkka) [inline restbot/pom.xml:54]: properties...

File: restbot/pom.xml [hunk 5]
```diff
@@ -18,21 +18,48 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.6</version>
```
Comment (patkkka) [inline restbot/pom.xml:59]: properties :)

**Review by mprtcz — APPROVED** [review]

**Review by patkkka — APPROVED** [review]

**Pull request merged**